sig-release/annual-report-2022.md

```diff
@@ -2,76 +2,105 @@
 
-   -
-   -
-   -
+1. What work did the SIG do this year that should be highlighted?
+   - [Migrated most of deb/rpm package building into release process to reduce Google Build Admin involvement in releases](https://github.com/kubernetes/release/issues/2737)
+   - Proof-of-concept of using OpenSUSE Build Service to build and publish packages using community infrastructure. Reflected in updates to [1731 - Publishing Kubernetes packages on community infrastructure](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/1731-publishing-packages). See [biweekly meeting from 18 October 2022](https://youtu.be/8l8X3vSAJAw?t=787) for a good overview.
+   - [Signing of Release Artifacts](https://github.com/kubernetes/enhancements/issues/3031).
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
-
-
+   - [Donation of a new project for SLSA Attestation](https://github.com/kubernetes-sigs/tejolote)
+   - We begun work on a new process for onboarding release manager associates and a ladder for becoming a full release manager.
 
 3. KEP work in 2022 (v1.24, v1.25, v1.26):
-  - alpha:
-    - [3000 - Artifact Distribution Policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3000-artifact-distribution) - v1.24
   - beta:
-    - [3031 - Signing release artifacts](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3031-signing-release-artifacts) - v1.25
-
+    - [3000 - Artifact Distribution Policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3000-artifact-distribution) - v1.25, v1.26
+    - [3027 - SLSA Level 3 Compliance](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3027-slsa-compliance) - v1.25
+    - [3031 - Signing release artifacts](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3031-signing-release-artifacts) - v1.25, v1.26
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
-
+   - [kubernetes-sigs/bom](https://github.com/kubernetes-sigs/bom/blob/main/OWNERS)
+   - [kubernetes-sigs/downloadkubernetes](https://github.com/kubernetes-sigs/downloadkubernetes/blob/master/OWNERS)
+   - [kubernetes-sigs/mdtoc](https://github.com/kubernetes-sigs/mdtoc/blob/master/OWNERS)
+   - [kubernetes-sigs/release-notes](https://github.com/kubernetes-sigs/release-notes/blob/master/OWNERS)
+   - [kubernetes-sigs/release-team-shadow-stats](https://github.com/kubernetes-sigs/release-team-shadow-stats/blob/master/OWNERS)
+   - [kubernetes-sigs/tejolote](https://github.com/kubernetes-sigs/tejolote/blob/main/OWNERS)
+   - [kubernetes-sigs/zeitgeist](https://github.com/kubernetes-sigs/zeitgeist/blob/master/OWNERS)
```

Comment:
Are there issues that can be linked which require attention? This will raise public awareness to the areas of each of these projects needing help.

Reply:
I'm not sure we should link the plain repositories here, we have a bunch of larger efforts tracked in keps as well as some good first ones. We probably need most help in graduating the enhancements and everything belonging to that.

Reply:
Updated!